app/controller/post.py

```
@@ -0,0 +1,72 @@
+    post_id: int,
+    service: PostService = Depends(get_post_service)
+):
+    post = service.get_post(post_id)
+    if not post:
+        raise HTTPException(status_code=404, detail="게시글을 찾을 수 없습니다")
+    return post
+
+@router.post("/", response_model=PostResponse)
+def create_post(
+    post_data: PostCreate,
+    service: PostService = Depends(get_post_service)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)